tracing/src/macros.rs

```diff
@@ -1959,6 +1959,52 @@ macro_rules! valueset {
             $next,
         )
     };
+
```

Comment:
Might be worth a comment here to separate the cases with literal field names from the cases with ident field names.

Reply:
done